Meter.c

```diff
@@ -302,6 +308,18 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
    }
    const size_t nValues = data->nValues;
 
+   const Machine* host = this->host;
+   if (!timercmp(&host->realtime, &(data->time), <)) {
+      int globalDelay = host->settings->delay;
+      struct timeval delay = { .tv_sec = globalDelay / 10, .tv_usec = (globalDelay % 10) * 100000L };
+      timeradd(&host->realtime, &delay, &(data->time));
+
+      for (size_t i = 0; i < nValues - 1; i++)
+         data->values[i] = data->values[i + 1];
+
+      data->values[nValues - 1] = sumPositiveValues(this->values, this->curItems);
+   }
+
```

Comment:
Maybe split this moved code segment as its own commit for clarity which of the changes go together.